lib/pinetwork.rb

```diff
@@ -185,28 +187,71 @@ def build_a2u_transaction(transaction_data)
     recipient = Stellar::KeyPair.from_address(transaction_data[:recipient])
     memo = Stellar::Memo.new(:memo_text, transaction_data[:identifier])
 
+    # Add time_bounds so we can place a time limit on the transaction and try the same
+    # one multiple times (in case of Horizon server errors)
+    now, timeout = Time.now.to_i, TX_SUBMISSION_TIMEOUT_SECONDS
+    max_time = now + timeout
+    time_bounds = Stellar::TimeBounds.new(min_time: now, max_time: max_time)
```

Comment:
```suggestion
    time_bounds = Stellar::TimeBounds.new(min_time:, max_time:)
```
To add to batch from comment above